xml/gl.xml

```diff
@@ -26206,7 +26201,6 @@ typedef unsigned int GLhandleARB;
             <param><ptype>GLenum</ptype> <name>format</name></param>
             <param><ptype>GLenum</ptype> <name>type</name></param>
             <param len="COMPSIZE(format,type,width,height,depth)">const void *<name>pixels</name></param>
-            <alias name="glTexSubImage3D"/>
```

Comment:
Please don't push anything else to this branch, while I report it to github and try to sort this out.